controllers/front/ProductController.php

```diff
@@ -999,6 +999,7 @@ private function getIdProductAttributeByRequestOrGroup()
      * if $checkedIdProductAttribute is available.
      * If not try to return the first available attribute, if none are available
```

Comment:
So this line is now false ?

Reply:
Nope, it still try to get an attribute if possible :thinking: